scripts/evaluate_release_tag.py

```diff
@@ -51,6 +55,11 @@ def should_build_docs(tag: Version) -> bool:
     latest_version = non_alpha_releases[-1]
     print(f"The latest non-alpha/rc/nightly tag is {latest_version}.")
 
+    non_beta_releases = filter_non_beta_relases(non_alpha_releases)
+    non_beta_releases.sort()
+    latest_version_non_beta = non_beta_releases[-1]
+    print(f"The latest non-beta tag is {latest_version_non_beta}.")
+
     previous_major = latest_version.major - 1
     previous_major_versions = [tag for tag in non_alpha_releases if tag.major == previous_major]
     previous_major_latest_version = previous_major_versions[-1]
```

Comment:
These lines look redundant now.

Reply:
They already were as `previous_major_latest_version` wasn't getting used anywhere, I'm not sure why is it there in the first place. I'll remove it